constraintsolver/include/GSolver.h

```diff
@@ -9,16 +9,18 @@
 #define GSOLVER_H_
 
 #include <AutoDiff.h>
+#include <engine/AlicaClock.h>
 
 #include <memory>
 #include <vector>
 
 #include <fstream>
 
 using namespace std;
+using alica::AlicaTime;
```

Comment:
not needed, you are referencing to AlicaTime and AlicaClock only in the namespace alica anyway